unitTests/ui/DynamicFacetManagerTest.ts

```diff
@@ -335,6 +335,21 @@ export function DynamicFacetManagerTest() {
         expect(collapsedFacets().indexOf(facets[2])).toBe(-1);
       });
 
+      it(`when there is an hidden facet (e.g. depends on another facet)
+      should not be taken into consideration when expanding/collapsing`, () => {
+        const dependantFacet = DynamicFacetTestUtils.createAdvancedFakeFacet({ field: '@dependantFacet', dependsOn: facets[0].options.id })
+          .cmp;
+        facets.unshift(dependantFacet);
+        const maximumNumberOfExpandedFacets = 2;
+        initForMaximumNumberOfExpandedFacets(maximumNumberOfExpandedFacets);
+        triggerQuerySuccess(queryManyFacetsResponse());
+
+        expect(hiddenFacets().indexOf(dependantFacet)).toBe(0);
+        expect(collapsedFacets().length).toBe(facets.length - (maximumNumberOfExpandedFacets + 1));
+        expect(collapsedFacets().indexOf(facets[1])).toBe(-1);
+        expect(collapsedFacets().indexOf(facets[2])).toBe(-1);
```

Comment:
Humm 🤔 

It seems there's 2 (or 3?) different check were doing in the same tests.

1) That the dependant facet is correctly visible
2) That the manager collapsed the correct number of facets
3) That the manager collapsed the facets in the correct order (?)

Should these by split in different UT with appropriate descriptive name ? 
Under the same "dependantFacet" UT block ?

Reply:
Removed the hidden check which wasn't relevant. Now we're only checking one thing: that the correct facets are expanded/collapsed.